docs/DATA_FORMATS_INPUT.md

```diff
@@ -115,6 +115,10 @@ For example, if you had this configuration:
 
   ## measurement name suffix (for separating different commands)
   name_suffix = "_mycollector"
+  
+  ## specifies if the incoming JSON data is an array of metric data (true) 
+  ## to parse or a single object (false)
+  array = false
```

Comment:
@johnrengelman I would recommend a very simple solution to detecting this....you can use the bytes package Index function, call `bytes.IndexByte(buf, '{')` and `bytes.IndexByte(buf, '[')`. If the index of `[` is lower than the index of `{`, then it's an array, otherwise it's a single document.